validation/src/collation.rs

```diff
@@ -315,6 +329,13 @@ impl Externalities {
 			});
 		}
 
+		if self.fees_charged != candidate.fees {
+			return Err(Error::FeesChargedInvalid {
+				expected: candidate.fees.clone(),
```

Comment:
```suggestion
				got: candidate.fees.clone(),
```

I think it should be the other way around.

Reply:
that is inconsistent with the other expected/got errors in the module.